core/src/test/java/org/apache/iceberg/TestBase.java

```diff
@@ -199,7 +201,7 @@ public class TestBase {
 
   @Parameters(name = "formatVersion = {0}")
   protected static List<Object> parameters() {
-    return Arrays.asList(1, 2, 3);
+    return Arrays.stream(ALL_VERSIONS).boxed().collect(Collectors.toUnmodifiableList());
```

Comment:
This looks overkill

Reply:
```java
Ints.asList(TestHelpers.ALL_VERSIONS);
```

Reply:
> ```java
> Ints.asList(TestHelpers.ALL_VERSIONS);
> ```

Great suggestion. I have implemented this approach.

Reply:
I just pushed another update.  I changed TestHelpers.ALL_VERSION from int[] to List<Integer>